Documentation/PackageDescription.md

```diff
@@ -439,13 +444,13 @@ www.semver.org.
 The major version signifies breaking changes to the API which requires
 updating existing clients. For example, renaming an existing type, removing
 a method or changing a method’s signature are considered breaking changes.
-This also includes any backwards incompatible bugfixes or behaviour changes
+This also includes any backwards incompatible bugfixes or behavior changes
 of existing API.
 
 #### The Minor Version
 
-Update the minor version if functionality is added in a backward compatible
-manner. For example, adding a new method or type without changing any other
+Increase the minor version if functionality is added in a backward compatible
```

Comment:
I would use "Increment" here instead of "Increase", as that's more typical language used when speaking about bumping version number components.

Reply:
That's a very good point, and I've changed it.